spec.bs

```diff
@@ -3172,9 +3172,10 @@ The <dfn for=Navigator method>getInterestGroupAdAuctionData(|config|)</dfn> meth
   1. Let |result| be a new {{AdAuctionData}}.
   1. Let |requestId| be the [=string representation=] of a [=version 4 UUID=].
   1. Set |result|'s {{AdAuctionData/requestId}} field to |requestId|.

```

Comment:
|result| is a dictionary {{AdAuctionData}}, so need to use map's set.
```suggestion
  1. [=map/Set=] |result|["{{AdAuctionData/requestId"}}] field to |requestId|.
```

Reply:
oh sorry, it needs to be ["{{AdAuctionData/requestId}}"]. Updated the suggestion.

Reply:
It should be fixed now.